IntroToCSharp/Pages/Lessons/Variables/TheTypeSystem.razor

```diff
@@ -0,0 +1,141 @@
+        <Keyword Word="data type" />s. C# has
+        <b>14</b> built in
+        <Keyword Word="primitive data type" />s that we use to
+        encode different types of data. This section of the book is a high level
+        reference of those 14
+        <Keyword Word="data type" />s.
+    </MudText>
+</Section>
+<Section Title="Integers">
+    <MudText>
+        There are 8 different integer types each of which is represented using a
+        different number of <Keyword Word="byte" />s as well as being <Keyword Word="signed"/> or
+        <Keyword Word="unsigned"/>.
+    </MudText>
+    <Note>You do not need to memorize this table but it is good to be aware.</Note>
+    <DataTable Headers="@intHeaders" Rows="@intRows"></DataTable>
+</Section>
+
+<Section Title="Characters">
+    <MudText>
+        In C#, the <Keyword Word="char"/> type represents a single character. This
+        is most often used when creating text using a <Keyword Word="string"/>. The
+        char data type is a special type of integer for which each integer value
+        represents a symbol to be displayed. The char data type uses 2 bytes of
+        data which means it has 65,536 distinct characters.
+    </MudText>
+    <Note>The characters that can be displayed with the </Note>
+
+    <MudText>
+        Unlike a string, a char is created by placing the character inside of
+        single quotes like this:
+    </MudText>
+    <CodeBlock AllowCopy>char someLetter = 'j';</CodeBlock>
+    <Note>You must place <b>exactly</b> on character between the single quotes.</Note>
+    <MudText>
+        Another interesting thing to note is that a char is not limited to the keys
+        on the keyboard. You can place any <Keyword Word="unicode"/> character between
+        the single quotes.
+    </MudText>
+    <CodeBlock AllowCopy ReplIt="JosephCollard/BallChar">
+char ball = '⚽';
+Console.WriteLine(ball);
+    </CodeBlock>
+    <MudText>
+        Each character has a hexidecimal <b>unicode</b> encoding associated with
+        it. If you happen to know the value (or search for one using your
+        favorite search engine), you can enter it using an <Keyword Word="escape
+        code"/>:
+    </MudText>
+    <CodeBlock AllowCopy ReplIt="JosephCollard/SmileyChar">
+char smiley = '\u263A';
+Console.WriteLine(smiley);
+</CodeBlock>
+    <Note>
+        When you use an escape code like the one above, you are only providing a
+        single character between the single quotes.
+    </Note>
+</Section>
+
+<Section Title="Floating-Point Numbers">
+    <MudText>
+        Integers are great for whole numbers, but what happens when we want to
+        store decimals or even repeating decimals like π? Floating-point numbers
+        are a data type that allows programmers to <b>approximate</b> a real number.
+    </MudText>
+    <MudElement Class="ma-4">
+        <DataTable Headers=@floatHeaders Rows=@floatRows/>
+    </MudElement>
+    <MudText>
```

Comment:
I would add in a line that mentions how the Literal indicators are used.

Reply:
You also could consider adding how to constrain to a certain amount of decimals places.

Reply:
I'm going to leave the constraints for another section on formatting text.

Reply:
What do you think about the additions I made to this sextion?

Reply:
I think that this is a good addition. The flow of this page feels a lot more fluid and clear now.